apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -843,10 +884,53 @@ int _wt_parse_dns(struct wt_options *opt, int argc, char *argv[])
 	return 0;
 }
 
-int _wt_parse_commands(struct wt_options *opt, int argc, char *argv[])
+int _wt_parse_connectbyrssi(struct wt_options* opt, int argc, char* argv[])
 {
-	int ret = 0;
-	wt_type_e options = _wt_get_opt(argc, argv);
+	if (argc < 6) {
+		return -1;
```

Comment:
better to put an error message for clarification, or print usage

Reply:
Done